test/utils_for_runtests.jl

```diff
@@ -73,15 +73,50 @@ end
 
 # TODO: docstring?
 function boundary_clustered_coord(N, L, x₀)
-    Δz(k)   = k < N / 2 + 1 ? 2 / (N - 1) * (k - 1) + 1 : - 2 / (N - 1) * (k - N) + 1 
-    z_faces = zeros(N+1) 
+    Δz(k)   = k < N / 2 + 1 ? 2 / (N - 1) * (k - 1) + 1 : - 2 / (N - 1) * (k - N) + 1
+    z_faces = zeros(N+1)
     for k = 2:N+1
         z_faces[k] = z_faces[k-1] + Δz(k-1)
     end
-    z_faces = z_faces ./ z_faces[end] .* L .+ x₀ 
+    z_faces = z_faces ./ z_faces[end] .* L .+ x₀
     return z_faces
 end
 
+function xspacings_field(grid, ℓ...)
+    Δx_op = xspacings(grid, ℓ...)
+    Δx_field = Field(Δx_op)
+    compute!(Δx_field)
+    return Δx_field
+end
```

Comment:
I'm ok to keep these for this PR, but as mentioned elsewhere the proper solution is to support the functions we need for testing. That way, tests will be much easier to understand and to refactor (and of course, also easier to write additional tests), plus the functionality will also be useful for users. We're trying to solve a problem that we've had in the past wherein test refactoring is extremely time consuming.

Reply:
Makes sense. I'll try overloading `Base.==` in which case we should be able to use that in the tests and get rid of these.

Reply:
`.==` worked out of the box so I ended up removing these definitions.